src/main/java/frc/robot/Cal.java

```diff
@@ -41,7 +41,7 @@ public static final class SwerveSubsystem {
      */
     public static final double SWERVE_FRONT_LEFT_ANGULAR_OFFSET_RAD = (2.0 * Math.PI) + 1.036;
 
-    public static final double SWERVE_FRONT_RIGHT_ANGULAR_OFFSET_RAD = (2.0 * Math.PI) + 1.005;
+    public static final double SWERVE_FRONT_RIGHT_ANGULAR_OFFSET_RAD = (1.0 * Math.PI) + 1.005;
```

Comment:
We should properly re-zero rather and set the offset back to the default rather than leaving it like this.

Reply:
Can we just merge so we can go live on main and fix it when we fix it?